### PR TITLE
imrovements to action variables

### PR DESCRIPTION
**Pull request by tim-thacker-nullify**
## Description

set defaults from github action variables where possible.
quote the inputs when they are passed to the cli tool.

## Linked Issues & PRs

- resolves <insert-issue-link>
- relates to <insert-pr-link>

[GitHub Issue Linking Docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

## Checklist

- [ ] I have added one of the `patch`, `minor`, `major` or `no-release` labels
- [ ] I have linked an issue from this repository using the **Development** option
- [ ] I have performed a self-review of my own code
- [ ] I have checked for redundant or commented out code
- [ ] I have commented my code where I can't make it self-documenting
- [ ] I have made corresponding changes to the documentation
- [ ] I have added any appropriate tests

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -7,47 +7,59 @@ inputs:
...
     - name: Execute Nullify DAST
       shell: sh
       run: |
-        ./nullify ${{ inputs.args }} \
-          --host ${{ inputs.host }} \
-          --github-token ${{ input.github-token }} \
+        repo_fullname=${{ inputs.github-repository }}
+        repo_owner=${repo_fullname%%/*}
+        repo_name=${repo_fullname#*/}
+        export GITHUB_ACTIONS=true
```
Comment (dstrates) [inline action.yml:54]: use gha env vars instead of exporting these

File: action.yml [hunk 2]
```diff
@@ -7,47 +7,59 @@ inputs:
...
+        download_url="https://github.com/Nullify-Platform/cli/releases/download/v${{ inputs.nullify-version }}/nullify_linux_amd64_${{ inputs.nullify-version }}"
+        wget -q $download_url -O ./nullify
+        sudo chmod +x ./nullify
     - name: Execute Nullify DAST
       shell: sh
       run: |
-        ./nullify ${{ inputs.args }} \
-          --host ${{ inputs.host }} \
-          --github-token ${{ input.github-token }} \
+        repo_fullname=${{ inputs.github-repository }}
```
Comment (dstrates) [inline action.yml:51]: why do we need repo_fullname and repo_name

File: action.yml (hunk 1 above)
Comment (tim-thacker-nullify) [reply action.yml:54]: they need to be exported, it didnt work without it

File: action.yml (hunk 2 above)
Comment (tim-thacker-nullify) [reply action.yml:51]: fullname comtains the owner and repo, we need to split it up

**Review by dstrates — APPROVED** [review]

**Pull request merged**